lib/ddtrace/environment.rb

```diff
@@ -6,15 +6,17 @@ module Environment
     # Defines helper methods for environment
     module Helpers
       def env_to_bool(var, default = nil)
-        ENV.key?(var) ? ENV[var].to_s.strip.downcase == 'true' : default
+        var = var.find { |env_var| ENV.key?(env_var) } if var.is_a?(Array)
+        var && ENV.key?(var) ? ENV[var].to_s.strip.downcase == 'true' : default
       end
 
       def env_to_int(var, default = nil)
         ENV.key?(var) ? ENV[var].to_i : default
       end
 
       def env_to_float(var, default = nil)
-        ENV.key?(var) ? ENV[var].to_f : default
+        var = var.find { |env_var| ENV.key?(env_var) } if var.is_a?(Array)
+        var && ENV.key?(var) ? ENV[var].to_f : default
       end
 
       def env_to_list(var, default = [])
```

Comment:
And to `env_to_list` too.
If you see a way to reuse code, go ahead, but don't sweat :)

Reply:
👍 updated